README.md

```diff
@@ -1,3 +1,5 @@
+# Baking with Friends README
```

Comment:
Change this readme to be indicative of what steps are needed to run it, not to "build" it.